m3u8/mixins.py

```diff
@@ -3,7 +3,7 @@
 from m3u8.parser import is_url
 
 try:
```

Comment:
I'd say kill this `as url_parser` stuff and just use `urllib.parse` in the actual code later